schema/defs-linux.json

```diff
@@ -93,7 +93,9 @@
             "type": "string"
         },
         "FileType": {
-            "type": "integer"
+            "description": "Type of a block or special character device",
+            "type": "string",
+            "pattern": "^[c|b|u|p]$"
```

Comment:
http://json-schema.org/latest/json-schema-validation.html#anchor33

On Wed, Apr 6, 2016 at 5:25 PM, W. Trevor King <notifications@github.com>
wrote:

> In schema/defs-linux.json
> <https://github.com/opencontainers/runtime-spec/pull/370#discussion_r58784803>
> :
>
> > @@ -93,7 +93,9 @@
> >              "type": "string"
> >          },
> >          "FileType": {
> > -            "type": "integer"
> > +            "description": "Type of a block or special character device",
> > +            "type": "string",
> > +            "pattern": "^[c|b|u|p]$"
>
> Is this JSON Schema following the RFC draft 4
> <https://tools.ietf.org/html/draft-zyp-json-schema-04>? I don't see
> anything about pattern in there, but would expect a regexp to look like
> ^[cbup]$.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/runtime-spec/pull/370/files/fc6670d4ae475faa20196488c87c373208a09223#r58784803>
>